docs/resources/firewalld.md.erb

```diff
@@ -0,0 +1,88 @@
+---
+title: About the firewalld Resource
+---
+
+# firewalld
+
+Use the `firewalld` InSpec audit resource to test rules set to match IP address's with hostnames.
+
+An firewalld rule specifies an IP address and what it's hostname is along with an aliase names it can have.
+
+## Syntax
+
+Use the where clause to match a property to one or more rules in the hosts file.
+
+    describe firewalld do
+      it { should be_running }
+      its ('default_zone') { should eq 'public' }
+      it { should have_service_enabled_in_zone('ssh', 'public') }
+      it { should have_rule_enabled('rule family=ipv4 source address=192.168.0.14 accept', 'public') }
+    end
+
+## Supported Properties
+
+### default_zone
+
+The `default_zone` property displays the default active zone to be used.
+
+    its ('default_zone') { should eq 'public' }
+
+### default_zone
+
+The `active_zones` property displays the open zone and all its open interfaces
+
+    its ('active_zones') { should eq "public\n  interfaces: enp0s3\n" }
```

Comment:
This seems like a bad user experience to me. A user shouldn't have to worry about including additional output when they're querying zones. `active_zones` should just return an array of names of zones that are active, right?

If we need to, we can always create a FilterTable of data for the zones... zone name, interface list, and state (active/inactive).

Reply:
I understand that, Yeah this is a good place for a FilterTable so the user can query for interfaces by zones.